pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java

```diff
@@ -46,14 +44,14 @@
 
 	@Column(name = "qty_on_sale")
 	private Integer onsale;
-
+	
 	@ManyToOne
 	@JoinColumn(name = "pizza_id")
-	private Pizza pizzaId;
+	private Pizza pizza;
 
 	@OneToMany
-	@JoinColumn(name = "pizza_variety_id")
-	private List<PizzaVariety> pizzaVarieties;
+	@JoinColumn(name = "id_pizza_variety")
```

Comment:
Isn't the column name id (from the pizza variety table) also use plural if the object is list/array so use pizzaVarieties